boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -562,16 +562,18 @@ pub unsafe fn main() {
     // IEEE 802.15.4 and UDP
     //--------------------------------------------------------------------------
 
-    let serial_num = nrf52840::ficr::FICR_INSTANCE.address();
-    let serial_num_bottom_16 = serial_num[0] as u16 + ((serial_num[1] as u16) << 8);
-    let src_mac_from_serial_num: MacAddress = MacAddress::Short(serial_num_bottom_16);
+    let device_id = nrf52840::ficr::FICR_INSTANCE.address();
+
+    let device_id_bottom_16: u16 = device_id[0] as u16 + ((device_id[1] as u16) << 8);
```

Comment:
```suggestion
    let device_id_bottom_16 = u16::from_le_bytes([device_id[0], device_id[1]]);
```
match the other boards / simpler?